src/main/java/compsci4039/week1/loops/Lab4Ex2.java

```diff
@@ -31,6 +31,8 @@ private static void calculatePrimeNumbersUntil(int number) {
     private static boolean isPrime(int number) {
         if (number < 0) {
             return false;
+        } else if (number == 1) {
+            return false;
```

Comment:
Hey @gsears!

Excellent catch. Indeed, I forgot about number 1. I would suggest modifying the first conditional clause tho, don't you think ```if(number < 2) return false``` would be more elegant?